# For some reason, they enable -msse2 on some systems. I prefer to use -xHost
# We disables -msse2 because it overrides -xHost.
# By Ward Poelmans <wpoely86@gmail.com>
--- CMakeLists.txt.orig	2013-11-08 16:18:59.000000000 +0100
+++ CMakeLists.txt	2013-11-08 16:19:22.000000000 +0100
@@ -343,12 +343,6 @@
   set(CMAKE_C_FLAGS_RELEASE "-O3 -ipo1 -DNDEBUG -Wl,-s")
   set(CMAKE_CXX_FLAGS_RELEASE "-O3 -ipo1 -DNDEBUG -Wl,-s")
 
-  # define WITH_SSE2 to enable the SSE2 instruction set.
-  # Available on Pentium 4, Athlon 64, and newer CPUs.
-  if(CMAKE_SIZEOF_VOID_P EQUAL 8 OR WITH_SSE2)
-    message(STATUS "Enabling the SSE2 instruction set")
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse2")
-  endif(CMAKE_SIZEOF_VOID_P EQUAL 8 OR WITH_SSE2)
 endif(${CMAKE_CXX_COMPILER_ID} MATCHES Intel AND UNIX)
 
 #Find if OpenMP support is enabled
